Add April 2026 newsletter (#2600)
writethedocs/www: docs/blog/newsletter-april-2026.rst

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: April 01, 2026
  :tags: newsletter

#########################################
Write the Docs Newsletter – April 2026
#########################################

Ahoy there, documentarians! Spring is beginning to pop up in these parts, which means growth, flowers, and of course pollen. I hope you are all able to find some pleasure around you that isn't covered in too much discomfort.

> In community news, the Berlin 2026 conference has been officially `announced </conf/berlin/2026/news/welcome/>`__! Start getting your ideas together for a proposal and book September 6–8 so we can meet up again. Looking forward to it already!

Reviewer: ```suggestion
In community news, the Portland 2026 conference has released `the full schedule </conf/portland/2026/news/announcing-schedule/>`__ -- in-person and virtual tickets are still available. The Berlin 2026 conference has been officially `announced </conf/berlin/2026/news/welcome/>`__! Start getting your ideas together for a proposal and book September 6–8 so we can meet up again. Looking forward to it already! 
```

Author: ```suggestion
In community news, the Portland 2026 conference has released `the full schedule </conf/portland/2026/news/announcing-schedule/>`__ -- in-person and virtual tickets are still available. And the Berlin 2026 conference has been officially `announced </conf/berlin/2026/news/welcome/>`__! Start getting your ideas together for a proposal and book September 6–8 so we can meet up again. Looking forward to it already!
```

Consider building your own solution if you have:

- Access to a AI coding agent,
- Dedicated engineering support,
- Either highly specialized/unconventional or very simple requirements,
- Technically proficient team, and
- Significant time (months, not days) available to implement

Use existing tools if there are:

- Limited engineering resources,
- Need for reliability and fast deployment,
- Standard documentation requirements, or
- Mandates to focus on timely content development

While building a custom solution may be appropriate under certain conditions, most contributors expressed concern about the long-term costs outweighing apparent gains.

See more Write the Docs resources about `choosing tools for documentation </guide/choosing-tools/>`__, `doc tools </topics/#doc-tools>`__, and `AI and LLMs <https://www.writethedocs.org/topics/#ai-and-llms>`__.

--------------------------------------
"Native" speaker requirements for jobs
--------------------------------------

> How often have you seen "*Native English speaker*" in a tech writer job ad? A recent Slack thread got our members thinking about what that requirement actually means, and whether it does more harm than good.

Reviewer 2: ```suggestion
How often have you seen "*Native English speaker preferred*" in a tech writer job ad? A recent Slack thread got our members thinking about what that requirement actually means, and whether it does more harm than good.
```

Author: ```suggestion
How often have you seen "*Native English speaker*" as a requirement in a tech writer job ad? A recent Slack thread got our members thinking about what that requirement actually means, and whether it does more harm than good.
```

Reviewer 2: Yes, better. Thanks @CollierCZ 👍

The concept of "native speakerism" inevitably creates the "non-native" opposite, and these binary labels don’t adequately reflect language identity, use, and understanding. In job ads, it seems to conflate two separate things: fluency and writing ability. The assumption being that if English is your first language, you are automatically a better writer.

> Many in the thread pushed back on this assumption, arguing that "non-native" speakers often bring more rigor to technical writing. Having had to consciously learn the language, they're less likely to make assumptions and more likely to question their choices. In technical writing, this instinct to question is a real asset, with one contributor noting that "doubt and second guessing are a key ingredient of authoring and editing." That said, one person noted that their documentation suite had accumulated stylistic quirks and phrasings over time from authors writing in their "non-native" language.

Reviewer 2: ```suggestion
Documentation written by "non-native speakers" can accumulate telltale signs over time: stylistic quirks, awkward phrasing, an overly formal register, or obscure vocabulary where a simpler word would do. Discussion in the thread presented the other perspective that "non-native" speakers often bring more rigor to technical writing. Having had to consciously learn the language, they're less likely to make assumptions and more likely to question their choices. In technical writing, this instinct to question is a real asset, with one contributor noting that "doubt and second guessing are a key ingredient of authoring and editing." 
```

Author: ```suggestion
Documentation written by "non-native" speakers can accumulate telltale signs over time: stylistic quirks, unusual phrasing, an overly formal register, or obscure vocabulary where a simpler word would do. Discussion in the thread included the other perspective that "non-native" speakers often bring more rigor to technical writing. Having had to consciously learn the language, they're less likely to make assumptions and more likely to question their choices. In technical writing, this instinct to question is a real asset, with one contributor noting that "doubt and second guessing are a key ingredient of authoring and editing."
```

Reviewer 2: Thank you @CollierCZ 👍

There’s also a legal dimension worth noting. In some jurisdictions, specifying "native English speaker" in a job ad may expose an employer to discrimination risk, especially when the requirement is about language ability not background.

If you see this requirement in an ad, it might be worth reaching out to query what recruiters actually mean by it. And if you see it in your own organization's listings, try to encourage use of "expert" or "highly competent" instead.

See more Write the Docs resources about `jobs and careers </topics/#jobs-and-careers>`__.

--------------------------------------
Docs without dedicated documentarians?
--------------------------------------

> As its capabilities continue to grow, some companies have positioned AI as a suitable replacement for human knowledge managers. In at least one instance, an entire writing staff was laid off as part of a “targeted adjustment” to the company’s overall strategy. WTD members how companies plan to implement AI systems without a dedicated team in place to manage them.

Author: ```suggestion
As its capabilities continue to grow, some companies have positioned AI as a suitable replacement for human knowledge managers. In at least one instance, an entire writing staff was laid off as part of a “targeted adjustment” to the company’s overall strategy. WTD members questioned how companies plan to implement AI systems without a dedicated team in place to manage them.
```

**Redefining the narrative**

Some stressed the necessity of connecting quality docs to AI performance to garner executive buy-in. The criteria for acceptable profit margins has shifted and companies are evaluating “revenue per seat” in their decisions to lay off employees. Investment in docs may increase company status in areas such as competitions (for example, for the best documentation).

**Moving forward**

The group was in consensus about these tenets: quality and customer service lags behind other priorities for some companies. Overarching decisions such as laying off an entire department are deeply impactful, but typically not a reflection of anyone’s individual performance. Prepare for the unexpected because your level of readiness in the face of uncertainty may be tested. At the same time, it’s more important than ever to evangelize docs, share wins, and find ways to tell the story of your value, so that company leadership can clearly understand the role tech comms play in delivering quality products.

> See more Write the Docs resources about `AI and LLMs </topics/#ai-and-llms>`__.

Author: ```suggestion
See more Write the Docs resources about `building docs culture </topics/#building-docs-culture>`__.
```
